source/plugins/idea.rst

```diff
@@ -1,10 +1,36 @@
-=============
+=======
 IntelliJ IDEA
-=============
+=======
 
+.. tip::
+    IntelliJ is recomended to use by the Sponge team, as it is a very smart IDE for beginners and advanced users.
```

Comment:
I agree, we recommend any IDE that sponge easily supports, and even then we don't tell people not to use other ones.